openlibrary/plugins/worksearch/code.py

```diff
@@ -395,6 +403,9 @@ def get_doc(doc: SolrDocument):
             )
             for ed in doc.get('editions', {}).get('docs', [])
         ],
+        ratings_average=doc.get('ratings_average', None),
+        ratings_count=doc.get('ratings_count', None),
+        want_to_read_count=doc.get('want_to_read_count', None),
```

Comment:
Here the second parameter `None` is not required as it will be the default to `doc.get`

e.g. `doc.get('want_to_read_count')` is the same as `doc.get('want_to_read_count', None)`